default/scripting/techs/defense/Network.focs.txt

```diff
@@ -84,7 +84,8 @@ EG_DEFENSE_NET
     ]
     stackinggroup = "DEFENSE_NET_STACK_@2@"
     priority = [[DEFAULT_PRIORITY]]
-    effects = SetMaxDefense value = Value + @1@ accountinglabel = "DEF_DEFENSE_NET_@2@"
+    effects = SetMaxDefense value = Value + @1@
```

Comment:
This stuff isn't a typo fix... you're making changes to the scripts that appear to remove specificity from the accounting labels...